content/arduino-cloud/02.features/00.security-considerations/security-considerations.md

```diff
@@ -0,0 +1,176 @@
+* Notify the Chief Information Security Officer, security team, and application owners of findings.
+* Conduct additional research as necessary to determine the extent of impact.
+
+If it is determined that a breach has occurred, system(s) or system component(s) may need to be taken offline until they can be locked down with additional security measures (change passwords and certificates, update firewall settings, etc.). An official statement will be issued, summarizing our findings and providing an estimated time frame for service restoration.
+
+### Data Retention and Disposal
+
+PII data will only be stored as long as Arduino legitimately needs it and in accordance with Arduino’s Privacy Policy.
+
+## Security Considerations for Specific Arduino Cloud Components
+
+### Arduino Cloud CLI
+
+The Arduino Cloud CLI provides a way for interacting with the Arduino IoT Cloud via the command line. The communication is carried out via an HTTPS connection and authentication is performed via API key credentials stored on the local machine.
+
+### Arduino Cloud Editor
+
+The Arduino Cloud Editor allows customers to create sketches they can subsequently deploy on boards. Each customer has a separate area where their sketches are stored so that no other customer has access to them. Sketches can be made publicly available to everyone, in which case access is granted by whomever possesses the link to access them.
+
+The Arduino Cloud Editor also provides a secret tab that can be used to securely store sensitive information needed by the device (e.g. WiFi credentials, API credentials). Secrets in this tab are encrypted with the AES-256 algorithm to safely store them in the Cloud.
+In order to communicate with the Arduino Cloud Editor, web browsers should be able to communicate over the internet to port 80, 443 and 8080 towards Arduino’s subdomains \*.arduino.cc.
+
+### Arduino IoT Cloud
+
+The Arduino IoT Cloud collects MQTT events through a data pipeline which is accessible only by the owner of the data and access authorization is enforced via our backend API.
+
+### Arduino Boards Security
+
+#### Device Identity and Secure Communication
+
+Devices onboarding and communication with the Arduino Cloud is ensured to protect the confidentiality of the data exchanged between Arduino devices and the Cloud. The following image shows the Device setup phase and the Device operation phase.
+
+![Device Setup & Operation Phases.](assets/device-phases.png)
+
+> **Note**: this communication process is available only on devices equipped with a secure element chip.
+
+**Device setup phase**: 
+* The setup phase is initiated by the web browser with a secure connection to the Arduino IoT Cloud over HTTPS requesting to create a new device.
+* The IoTCloud provides a special sketch that is run on the device to generate:
```

Comment:
Is this supposed to be "IoT Cloud" @rhaidiz? With a space in between `IoT` and `Cloud`´?

Reply:
I believe so as we write it that way also in the [docs](https://docs.arduino.cc/arduino-cloud/).